src/systems/filecoin_mining/storage_proving/storage_proving_subsystem.id

```diff
@@ -24,4 +24,9 @@ type StorageProvingSubsystem struct {
         challengeSeed     sector.PoStRandomness
         challengeTickets  [sector.ChallengeTicket]
     ) sector.PoStProof
+
+    ValidatePoStProof(
```

Comment:
Having `ValidatePoStProof` here is even more confusing when `post_generator.id` contains `VerifyPoStProof`. I'm going to assume this is an oversight and not intentional. If intentional, please explain.

Reply:
sorry. yes it is. fixed